scripts/publish.sh

```diff
@@ -313,3 +359,34 @@ if [ "${1}" = "rollback-ssm" ]; then
 		rollback_ssm ${bahrain_region}
 	fi
 fi
+
+# Publish using CI/CD pipeline
+# Following scripts will be called only from the CI/CD pipeline
+if [ "${1}" = "cicd-publish" ]; then
+	if [ "${2}" = "dockerhub" ]; then
+		publish_to_docker_hub amazon/aws-for-fluent-bit:latest amazon/aws-for-fluent-bit:${AWS_FOR_FLUENT_BIT_VERSION}
+	elif [ "${2}" = "aws-us-gov" ]; then
+		for region in ${gov_regions}; do
+			gov_cloud_sync ${region} ${gov_regions_account_id} 
```

Comment:
`gov_cloud_sync` isn't in this PR right?

Reply:
Yes, you are right. As we talked offline, I will send the gov cloud changes in the next PR. In this PR, this block will never be executed.